src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/entity/AdditionalCondition.kt

```diff
@@ -43,7 +43,7 @@ data class AdditionalCondition(
   )
   @Fetch(FetchMode.SUBSELECT)
   @OrderBy("dataSequence")
-  val additionalConditionData: List<AdditionalConditionData> = emptyList(),
+  val additionalConditionData: MutableList<AdditionalConditionData> = mutableListOf(),
 
```

Comment:
These values were never immutable by design,.. in the old code we used copy to add the the condition data, 

```
    val updatedAdditionalCondition = additionalCondition.copy(
      conditionVersion = version,
      additionalConditionData = additionalConditionData,
```
now we have removed copy as a method to updating the entity we must change the list to mutable its true state.